<launch>
  <node pkg="rosbag" type="play" name="rosbag" args="-d 2 $(find buoys)/1.bag"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/mv_bluefox_camera_node/image_raw"/>
  </node>
</launch>
